### PR TITLE
Whats insp report v2

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Resources/WhatsInInspectionReportResource.cs [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-
-        public string OurUnitOfMeasure { get; set; }
-
-        public decimal QtyInStock { get; set; }
-
-        public decimal QtyInInspection { get; set; }
+        public IEnumerable<WhatsInInspectionReportEntryResource> PartsInInspection { get; set; }
 
-        public ReportReturnResource OrdersBreakdown { get; set; }
+        public ReportReturnResource BackOrderData { get; set; }
```
Comment (richardiphillips) [inline src/Resources/WhatsInInspectionReportResource.cs:11]: are they really back orders?

**Review by richardiphillips — APPROVED** [review]
phew complicated

File: src/Resources/WhatsInInspectionReportResource.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Resources/WhatsInInspectionReportResource.cs:11]: I'm not sure really. This little ResultsModel/ReportReturnResource is just a copy of the data in this footer of the old report
Screenshot 2022-04-21 092248

Do you think there's a better name for it?

Comment (richardiphillips) [reply src/Resources/WhatsInInspectionReportResource.cs:11]: no fair enough. perfect name

**Pull request merged**